src/lib.rs

```diff
@@ -34,6 +34,8 @@ The following feature flags enable additional functionality for this crate:
 #![cfg_attr(docsrs, feature(doc_notable_trait))]
 #![deny(missing_docs)]
 
+pub use arrow2;
```

Comment:
```suggestion
/// Re-export of the arrow2 crate depended on by this crate.
///
/// We recommend that you use this re-export rather than depending on arrow2
/// directly to ensure compatibility; otherwise, rustc/cargo may emit mysterious
/// error messages.
pub use arrow2;
```